src/modules/esl-popup/core/esl-popup.ts

```diff
@@ -106,8 +109,16 @@ export class ESLPopup extends ESLToggleable {
 
   /** Container element that define bounds of popups visibility */
   @memoize()
-  protected get $container(): HTMLElement | null {
-    return this.container ? TraversingQuery.first(this.container, this) as HTMLElement : null;
+  protected get $container(): HTMLElement | undefined {
+    return this.container ? TraversingQuery.first(this.container, this) as HTMLElement : this._containerEl;
+  }
```

Comment:
Is that right that the `$container` is memoized while `containerEl` could be passed dynamically via params?

Reply:
Memoization of $container (and as a three another getters) exist just only during popups update loop. 
It creates after show and clears on hide.
Nothing can open popup again if it is in the open state (if I understand right the design of toggleable).